optuna/importance/_fanova/_tree.py

```diff
@@ -136,8 +136,8 @@ def _get_marginalized_statistics(self, feature_vector: numpy.ndarray) -> Tuple[f
             node_indices.append(node_index)
             active_features_cardinalities.append(_get_cardinality(search_spaces))
 
-        node_indices = numpy.array(node_indices, dtype=numpy.int32)
-        active_features_cardinalities = numpy.array(active_features_cardinalities)
+        node_indices = numpy.array(node_indices, dtype=numpy.int32).tolist()
+        active_features_cardinalities = numpy.array(active_features_cardinalities).tolist()
```

Comment:
Seems like unnecessary conversions are taking place here. Should we give them new variable names or just skip these lines instead?